here-naksha-lib-psql/src/main/java/com/here/naksha/lib/psql/PsqlStorage.java

```diff
@@ -545,27 +554,15 @@ public boolean pg_plv8() {
     }
   }
 
-  @Override
-  public void initStorage() {
-    initStorage(null);
-  }
-
   /**
    * Ensure that the administration tables exists, and the Naksha extension script installed in the latest version.
    *
-   * @param params Parameters special to PostgresQL.
    * @throws SQLException If any error occurred while accessing the database.
    * @throws IOException  If reading the SQL extensions from the resources fail.
    */
   @Override
-  public void initStorage(@Nullable Map<String, Object> params) {
```

Comment:
As a quick check I executed tests on local and found several Hub tests failing with this exception:
```
Caused by: java.lang.NullPointerException: Cannot invoke "com.here.naksha.lib.psql.PsqlStorage$Params.pg_plv8()" because "this.params" is null
	at com.here.naksha.lib.psql.PostgresStorage.initStorage(PostgresStorage.java:429)
	at com.here.naksha.lib.psql.PsqlStorage.initStorage(PsqlStorage.java:565)
	at com.here.naksha.lib.core.storage.IStorage.initStorage(IStorage.java:58)
	at com.here.naksha.lib.hub.storages.NHAdminStorage.initStorage(NHAdminStorage.java:82)
	at com.here.naksha.lib.hub.NakshaHub.storageSetup(NakshaHub.java:115)
	at com.here.naksha.lib.hub.NakshaHub.<init>(NakshaHub.java:95)
```

Reply:
I understand your point. I've yet to run the tests successfully locally for this project, on any branch.

I will let @xeus2001 turn the CI tests on when he returns.

Reply:
I've fixed the NPE anyway.

Reply:
Ok, few comments here:
1. "enabling" pipeline execution on a branch doesn't require Alex's access , it can be done straightaway by adding branch name into the pipeline file that I shared earlier. Or did you mean "decision" to enable, then I get it (though no harm in enabling it).
2. With respect to the change to this file, I see that `initStorage(param)` method is removed. This was intentionally overridden from `IStorage` to support custom scenario (like location execution) where we want to explicitly disable `pg_hint` plan by providing it as part of `params`. Due to removal of this method, the scenario no longer works and tests fail. So, my question is - is that an intentional change? If no, then it should be reverted. If it was intentional, and we want all to use `withParams` method from now on, then `IStorage` should also have this method declared, as everywhere we are coding against `IStorage` interface.

Reply:
1. Ok, I've enabled the CI pipeline for alw_jbon branch.
2. Will try to address this now.